paymentsheet/src/test/java/com/stripe/android/paymentelement/embedded/SharedPaymentElementViewModelTest.kt

```diff
@@ -33,30 +36,53 @@ import kotlin.test.Test
 @ExperimentalEmbeddedPaymentElementApi
 @RunWith(RobolectricTestRunner::class)
 internal class SharedPaymentElementViewModelTest {
+    private val defaultConfiguration = EmbeddedPaymentElement.Configuration.Builder("Example, Inc.").build()
```

Comment:
I refactored these tests here, so no changes to tests outside of this refactor, and the one new test.